services/gateway-api/src/circuit_breaker/health_monitor.py

```
@@ -0,0 +1,450 @@
+
+        except Exception as e:
+            result.status = HealthStatus.UNHEALTHY
+            result.error = str(e)
+            result.response_time_ms = (time.time() - start_time) * 1000
+
+        # Update metrics
+        self._update_metrics(service_name, result)
+
+        return result
+
+    def _update_metrics(self, service_name: str, result: HealthCheckResult):
```

Ruff (0.14.8)

Missing return type annotation for private function `_update_metrics`

Add return type annotation: `None`

(ANN202)